Source/driver/check_for_nan.f90

```diff
@@ -0,0 +1,29 @@
+module nan_check
```

Comment:
This already exists in AMReX, do we need to duplicate it in Castro?

Reply:
oh, guess not.  Where is it in AMReX?  I was probably going to move it out anyway one I am happy with the debugging, but it is invaluable for debugging.

Reply:
It's in the FArrayBox class, it's implemented in C++ (contains_nan).

Reply:
right, but this is not a FAB at this point.  This is called from within the Fortran itself, when it is just an array.  That's where I needed it.